gramex/data.py

```diff
@@ -1471,6 +1492,7 @@ def _filter_mongodb(url, controls, args, database=None, collection=None, query=N
         for col, val in data.iloc[0].iteritems():
             if type(val) in {bson.objectid.ObjectId}:
                 data[col] = data[col].map(str)
+
```

Comment:
Let's add test cases in test_lib/test_data.py.

But manually also test for

- Filter by ID
- Filter a table that has multiple ID columns using two IDs
- Filter a table with a boolean, date, int, float and str column with EACH of these
- Delete by ID, date, boolean, int, str
- Update one or more fields filtered by ID, date, boolean, int, str
- Test Gramex Admin with a MongoDB backend: add user/update password/delete users